modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/js/ClaySampleToastAlert.js

```diff
@@ -37,6 +37,16 @@ export default function ClaySampleToastAlert() {
 		});
 	};
 
+	const onClick5Seconds = () => {
+		openToast({
+			autoClose: 5000,
```

Comment:
I would remove this autoClose line 42. I think we want to check the default openToast auto closes without directly setting the autoClose API. Would the test pass if we remove this line?

Reply:
cc/ @matuzalemsteles could you verify that 5 seconds is default case for toast message? Or do we need to explicitly set an autoClose value?

Reply:
done ✅
I removed the autoClose line and the test worked as expected.